### PR TITLE
doc: workspace-structure

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: book/src/workspace-structure.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Dist has two types of configuration:
+
+- **Workspace**:
+- **Package**: 
+
+The vast majority of configuration for dist is workspace configuration. Because
+we understand language-specific config manifests like `Cargo.toml` and
+`package.json`, package configuration ususally does not require any additional
+files.
```
Comment (mistydemeo) [inline book/src/workspace-structure.md:15]: Worth explicitly mentioning the "other language" support that does?

File: book/src/workspace-structure.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+## Workspace configuration
+
+A workspace file may be:
+
+* `dist-workspace.toml`
+* (deprecated in 0.23.0) `Cargo.toml` (Rust users only, goes in `[workspace.metadata.dist]`)
+
+### Specifying The Members Of Your Workspace
+
+Create a `workspace.members` field in your `dist-workspace.toml` that points to
```
Comment (mistydemeo) [inline book/src/workspace-structure.md:26]: I wonder if we want to link to a live example of this kind of repo? https://github.com/axodotdev/axolotlsay-hybrid

File: book/src/workspace-structure.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* `package.json` (for an npm package)
+
+In the case of a `Cargo.toml` and `package.json`, we'll do our best to find basic package
+info like package name, version, repository, binaries among the native language-specific config.
+
+However these files do not natively support dist-specific config, so you may 
+need to place a dist config *next* to them to specify additional dist-specific
+config. If we see the language-specific package file, we will automatically
+check for a neighbouring dist config and merge its contents in. If you wish to
+"shadow" or change a value that is present in your package manifest- you can
```
Comment (mistydemeo) [inline book/src/workspace-structure.md:60]: Nitpick - we probably want to pick a single syntax for dashes and stick to it. These aren't rendering the same as each other.

File: book/src/workspaces/structure.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ contains a single package.
 
 Dist has two types of configuration:
 
-- **Workspace**:
-- **Package**: 
+- **Workspace**
```
Comment (ashleygwilliams) [inline book/src/workspaces/structure.md:11]: i was going to add brief descriptions of these next to the words- do you think it needs that?

Comment (mistydemeo) [reply book/src/workspaces/structure.md:11]: I think it's a good call.

**Review by mistydemeo — APPROVED** [review]

**Pull request merged**